ingress/controllers/nginx/controller.go

```diff
@@ -747,8 +921,11 @@ func (lbc *loadBalancerController) Run() {
 	go lbc.endpController.Run(lbc.stopCh)
 	go lbc.svcController.Run(lbc.stopCh)
 
+	time.Sleep(1 * time.Second)
```

Comment:
to avoid N "deferring sync till endpoints control" errors before the initial sync